tests/System/transformationSystem.sh

```diff
@@ -42,10 +42,11 @@ if [ -d "TransformationSystemTest" ]; then
 fi
 echo "Creating TransformationSystemTest"
 mkdir -p TransformationSystemTest
-directory=/lhcb/certification/Test/INIT/$version/$tdate/$stime
+directory=/dteam/DIRAC/certification/Test/INIT/$version/$tdate/$stime
 #selecting a random USER Storage Element
 #SEs=$(dirac-dms-show-se-status |grep USER |grep -v 'Banned\|Degraded\|-2' | awk '{print $1}')
-SEs=$(dirac-dms-show-se-status |grep BUFFER |grep -v 'Banned\|Degraded\|-new' | awk '{print $1}')
+#get all SEs that are active
+SEs=$(dirac-dms-show-se-status  |grep -v 'Banned\|Degraded\|-new' | awk '{print $1}')
```

Comment:
Should this just be "RAL-SE" ?

Reply:
leave it, it's fine.

Reply:
When I submitted my test I restricted to RAL-SE, because I saw also LogSE and SandboxSE in this list
```
dirac-dms-show-se-status
False False dteam
SE        ReadAccess WriteAccess RemoveAccess CheckAccess
=========================================================
CESNET-SE Active     Active      Active       Active
LogSE     Active     Active      Active       Active
RAL-SE    Active     Active      Active       Active
SandboxSE Active     Active      Active       Active
```
Maybe ``grep "-SE "`` ?

Reply:
you also want to remove the `Probing` status ?

Reply:
Actually: do this:
`SEs=$(dirac-dms-show-se-status  |grep -v 'Banned\|Probing\|-new' | awk '{print $1}')`

Reply:
So LogSE or SandboxSE would be fine? (Just checking again).

Reply:
No they would not, of course. Just grep `-SE` as you were suggesting.

Reply:
Ok, done